safe/metadata/exposure_layer_metadata.py

```diff
@@ -127,6 +127,13 @@ class ExposureLayerMetadata(GenericLayerMetadata):
             'inasafe/'
             'value_map/'
             'gco:Dictionary'),
+        'minimum_needs': (
```

Comment:
Ah, I forgot to delete this. It's one of the things I want to ask in standup.
Should we let minimum needs expressed in exposure keywords too, or just taken from the profile as the only way to get it?